### ISSUE TITLE
Childbrowser setup instructions

### ISSUE THREAD

**Issue by joehinder** [purplecabbage/phonegap-plugins#43]
Looks like the Readme.txt no longer contains the installation instructions for the Childbrowser plugin. Think you all can put them back in?

**Comment by brunomikoski** [comment]
Yeah, I tried everything i could to install Childbrowser on WP7 without success, if anyone know what I can do I'll be very thankfull

**Comment by whirlwin** [comment]
Yes. Instructions would be awesome!

**Comment by vaibhavbparikh** [comment]
for WP7 ,am not able to install Childbrowser plug in, i tried every thing but no success...!!

if any one implemented and can help or author him self can provide readme then it would be great help .

**Comment by whirlwin** [comment]
After a few tries got it up and running. [Here](https://github.com/niths/niths-wp7-fadderuke-app) is my repository. I have made changes to the native C# file, the cordova.js file and the childbrowser.js file.

**Comment by vaibhavbparikh** [comment]
thanks mate ...it would be grate help for all folks

**Comment by ciaran036** [comment]
I made a call like this in JavaScript: ChildBrowser.prototype.showWebPage(result, false);

But I've been getting the error: "Error in error callback: ChildBrowserCommand594398001 = SyntaxError: Invalid character". 

The URL I've been entering is http://www.google.com 

The child browser doesn't initiate. 

By the way, as for actually integrating the plugin into your project, I think all you have to do is to copy the ChildBrowserCommand.cs to your Plugins folder, and then copy the ChildBrowser.js to the www folder.
